### PR TITLE
Add warning of ticket fee errors to close wallet action

### LINKED ISSUE #3025: [1.6.0-rc4] Add warning of ticket fee errors to close wallet action
If the wallet that has tried to purchase a ticket but ended up with a fee error for whatever reason, when trying to close Decrediton, a modal is displayed indicating there are still tickets with the fee unpaid.

However, when trying to close the wallet by way of Settings -> Close Wallet, this warning is **not** displayed.

Ideally, it should also be displayed there.

### LINKED ISSUE #3043: [1.6-rc4] Improve ticket fee error modal
Right now this is the modal we are showing when decrediton still has ticket with fee errors:

image

From @davecgh on matrix:
~~`perhaps: "You still have unpaid ticket fees." For the first part. The second one I'm not entirely sure what it means by "re-sync fees" to offer a better suggestion, but it would at the very least be "Click on this button to re-sync fees".
The title should also be "VSP Ticket Fee Error" imo.  It may or may not be more than one ticket.`~~

~~We also need to change the `confirm` button to `close decrediton`~~

image

Note that: if changing it to `close anyway`, we need to change our other modals, for keeping consistency

### PR DESCRIPTION
**Pull request by bgptr**
Closes #3025
Closes #3043

### CODE REVIEW

**Review by amassarwi — APPROVED** [review]

File: app/components/views/SettingsPage/SettingsPage.jsx [hunk 1]
```diff
@@ -25,10 +25,19 @@ const closeWalletWithAutobuyerModal = (walletName) => (
   />
 );
 
+const closeWalletWithUnpaidFeeModal = (walletName) => (
+  <T
+    id="settings.closeWalletModalWithUnpaidFeeModal"
+    m="Are you sure you want to close {walletName} and return to the launcher? You still have unpaid tickets fee. If you proceed and they are chosen to vote, they will be missed."
```
Comment (jholdstock) [inline app/components/views/SettingsPage/SettingsPage.jsx:31]: We probably need some more information here - what should the user do if they see this modal?

- Do they need to press a button somewhere?
- Do they just need to wait a bit longer? - 1 minute? Wait for a block to be mined?

**Review by vctt94 — COMMENTED** [review]
As jhold already comment about the message here, I think maybe we can also close https://github.com/decred/decrediton/issues/3043 ? 

What do you guys think

File: app/components/views/SettingsPage/SettingsPage.jsx (hunk 1 above)
Comment (alexlyp) [reply app/components/views/SettingsPage/SettingsPage.jsx:31]: Good point, Let's change it to the following.

"Please go back to the ticket status page and press the Sync VSP Tickets button to ensure that all of your ticket fees are paid."

File: app/components/views/SettingsPage/SettingsPage.jsx [hunk 2]
```diff
@@ -17,19 +17,7 @@ const closeWalletModalContent = (walletName) => (
   />
 );
 
-const closeWalletWithAutobuyerModal = (walletName) => (
-  <T
-    id="settings.closeWalletModalWithAutobuyerModal"
```
Comment (jholdstock) [inline app/components/views/SettingsPage/SettingsPage.jsx:22]: I dont think this should have been removed. I am trying to close a wallet with the ticketbuyer running, and I see the action being logged
```
 action     {type: "SHOW_AUTOBUYER_RUNNING_MODAL"}
```
but nothing happens at all. I am not getting any popup modal, and decrediton does not close.

**Review by jholdstock — APPROVED** [review]
Tested functionality and its looking good

**Review by alexlyp — APPROVED** [review]

**Pull request merged**